build_tools/github_actions/runner/validate_trigger.releaser.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+set -euo pipefail
+
+SCRIPT_DIR="$(dirname -- "$( readlink -f -- "$0"; )")";
+
+source "${SCRIPT_DIR}/functions.sh"
+
+ALLOWED_EVENTS=(
+  "push"
```

Comment:
does 'releaser' need the `push` event?

Reply:
It doesn't seem like an unreasonable thing to trigger on and it's something that's limited to the main repository. I could remove it

Reply:
Ok fair. YAGNI and all that